manifests/x/xushengfeng/eSearch/1.6.0/xushengfeng.eSearch.locale.zh-CN.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NLSU.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: xushengfeng.eSearch
+PackageVersion: 1.6.0
+PackageLocale: zh-CN
+Publisher: xushengfeng
```

```suggestion
Publisher: xsf
```
This must match Control Panel.